.github/workflows/examples.yml

```
@@ -0,0 +1,88 @@
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - run: |
+          python -m pip install --upgrade pip
+          pip install urbackend==0.1.1 python-dotenv==1.2.2 pytest==9.1.1
+      - run: pytest tests/ -v
+</write_to_file></tool_call>
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)